.github/workflows/live-protection.yml

```
@@ -0,0 +1,16 @@
+on: [pull_request_target]
+
+jobs:
+  comment:
+    if: ${{ github.event.issue.user.login != 'cxwtool' && github.ref == 'refs/heads/live' }}
```

```suggestion
    if: ${{ && github.base_ref == 'refs/heads/live' && github.head_ref != 'refs/heads/main' }}
```

I think it's good to check that the source branch is main.

---

@gewarren The tool creates a "snapshot" branch named `publish-ID`. So checking `main` doesn't seem correct.

---

For the live check, I'm not sure if I should be using `base_ref` or `ref`, or if there are any differences.

---

Oh right. How about:

```suggestion
    if: ${{ github.base_ref == 'refs/heads/live' && !(github.event.issue.user.login == 'cxwtool' || github.head_ref == 'refs/heads/main') }}
```

---

> For the live check, I'm not sure if I should be using `base_ref` or `ref`, or if there are any differences.

`base_ref` seems to be more specific to pull requests, so I'd go with that. `ref` can be a branch or tag.

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context